test/test_calibration.py

```diff
@@ -0,0 +1,80 @@
+import pytest
```

Comment:
One issue here, is that we've been using the unittest module and not pytest for unit tests. This would pull in another build dependency. @ColmTalbot Any chance this can be converted to unittest, or a good reason we should move to pytest instead?

Reply:
Otherwise, I think this is ready to go.

Reply:
@ahnitz We've been using `pytest` in `gwin`... @duncanmmacleod decided to use pytest instead of unittest, I think because it has some better features. Since @ColmTalbot originally wrote this in `gwin`, the tests he wrote were in `pytest`. Would it be so bad to support both?